script.kodi.android.update/default.py

```diff
@@ -16,7 +16,7 @@
 # -*- coding: utf-8 -*-
 
 import os, time, datetime, traceback, re
-import urllib, urllib2, socket
+import urllib, urllib2, socket, json
```

Comment:
I can accept this as is but note that the addon is not python3 compatible (urllib2 is split across several modules in py3)

Reply:
fixing now

Reply:
@enen92 should be good now.